coalib/processes/Processing.py

```diff
@@ -250,8 +251,7 @@ def get_file_dict(filename_list, log_printer):
     file_dict = {}
     for filename in filename_list:
         try:
-            with open(filename, "r", encoding="utf-8") as _file:
-                file_dict[filename] = tuple(_file.readlines())
+            file_dict[filename] = Fileproxy(filename)
```

Comment:
Here's the issue. Why file_dict here ?
Just `fileproxy_set` or `file_set` is enough right ?